app/Http/Controllers/Bookcal/CalendarBookcal.php

```
@@ -2,16 +2,308 @@
+        return view('bookcal.calendar_edit', [
+            'object' => $object,
+            'extrafields' => $extrafields,
+            'form' => $form,
+            'langs' => $langs,
+            'backtopage' => $backtopage,
+            'backtopageforcancel' => $backtopageforcancel,
+            'permissiontoadd' => $permissiontoadd,
+        ]);
+    }
+    
+    private function update(Request $request, int $id): RedirectResponse
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$request'. (undefined)

(UnusedFormalParameter)